docs/source/models/gbd2017/causes/lower_respiratory_infections/index.rst

```diff
@@ -17,12 +17,29 @@ syncytial virus (*RCV*). LRI can be caused by multiple pathogens and the pathoge
 Pneumococcal pneumonia is the largest cause of LRI
 mortality. [Wikipedia]_, [GBD-2017-YLD-Capstone-Appendix-1]_
 
+Transmission of LRI may occur via several pathways, including direct physical contact,
+fomites, direct droplet spread, and suspended small particles. Intermingling of
+large numbers of people can facilitate transmission of respiratory pathogens. [CDC]_
+
+Malnutrition was identified as a leading risk factor for lower respiratory infection
```

Comment:
Maybe preface this with "In GBD 2016, malnutrition was identified..." to make it clear what research this is referring to.